### ISSUE TITLE
Current day selected on keyboard navigation

### ISSUE THREAD

**Issue by vatsalyaagarwal** [Hacker0x01/react-datepicker#2705]
**Is your feature request related to a problem? Please describe.**
From keyboard navigation when you reach the day selector it goes to the current date, there should be an option to start it from the min day of every month

**Describe the solution you'd like**
option to have the min day for every month selected on day picker navigation

**Describe alternatives you've considered**
adding min day as 1 and adding all the other days in the current month to excluded days
adding keyboard selected class and tab index to the day i want it to go to

**Additional context**
